herd/libdir/aarch64deps.cat

```diff
@@ -62,30 +65,29 @@ let rec dtrm =
 let ADDR = Rreg \ DATA
 let basic-dep =
    [Exp & R | Rreg]; dtrm?
-let data = (basic-dep; [DATA]; iico_data+; [W]) \ same-instance
-let addr = [Exp & M]; (basic-dep; [ADDR]; iico_data+; [M | TLBI | IC.IVAU | DC.CVAU]) \ same-instance
-let ctrl = (basic-dep; [BCC]; po) \ same-instance
+let data = [Exp & R]; (basic-dep; [DATA]; iico_data+; [W]) & ~same-instance
+let addr = [Exp & R]; (basic-dep; [ADDR]; iico_data+; [M | TLBI | IC.IVAU | DC.CVAU]) & ~same-instance
+let ctrl = [Exp & R]; basic-dep; [BCC]; po
```

Comment:
I dropped `~same-instance` from the control dependency because the definition includes a `po` edge. There is no way for `ctrl` to connect to effects internal to one instruction instance. That said, it can be returned for presentation reasons.

Reply:
Good point, I wonder if we could require a lrs to avoid the ~same-instance in data and addr, but let's leave that for now.